### DISCUSSION TITLE
Bidentate Sticking Coefficients are incorrect

### DISCUSSION THREAD

**Discussion by ChrisBNEU** [ReactionMechanismGenerator/RMG-Py#2530] [General]
I came across this comment in [reacton.py](https://github.com/ReactionMechanismGenerator/RMG-Py/blob/8ba4c0dd3baa02d5541538d11c875d7ca9d59339/rmgpy/reaction.py#L768).
```
# ToDo: missing the sigma terms for bidentate species. only works for single site adsorption
```
 I think it is referring to the sigma shown here: 
<img width="311" alt="image" src="https://github.com/ReactionMechanismGenerator/RMG-Py/assets/56306881/94f296a6-2360-4515-8966-bbd1473d6ae2">

As far as I can tell, this comment is still correct. I don't think we have any families that would be affected by this, but I am bringing it up since we may be creating a lot more families soon. I think the the situation we would need it is if we have a gas phase species colliding with a bidentate species. The only reason we haven't added it in is because we haven't had a reaction to test it on. 

Source for equation in pp 39, equation 56 in chemkin surface manual. 
[chemkin11surfacechemkin.pdf](https://github.com/ReactionMechanismGenerator/RMG-Py/files/12370978/chemkin11surfacechemkin.pdf)

**Comment by ChrisBNEU** [comment]
The reason I made a discussion is because I want to know: 
- would excluding the sigma term cause any of our current families rates to be wrong? 
- if it doesn't currently affect anything, should we try to fix this now or just wait till we need it for a new family?

**Reply by JacksonBurns** [reply]
Do we have a way to test it without already having a new family that would require it?

**Comment by bjkreitz** [comment]
I made a PR to fix this issue

**Reply by JacksonBurns** [reply]
See #2533
